pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/UnusedPrivateFieldRule.java

```diff
@@ -27,9 +28,21 @@
 import net.sourceforge.pmd.lang.java.symboltable.VariableNameDeclaration;
 import net.sourceforge.pmd.lang.symboltable.NameDeclaration;
 import net.sourceforge.pmd.lang.symboltable.NameOccurrence;
+import net.sourceforge.pmd.properties.PropertyDescriptor;
+import net.sourceforge.pmd.properties.PropertyFactory;
 
 public class UnusedPrivateFieldRule extends AbstractLombokAwareRule {
 
+    private static final PropertyDescriptor<List<String>> IGNORED_FIELD_NAMES =
+                PropertyFactory.stringListProperty("ignoredFieldNames")
+                    .defaultValue(Arrays.asList("serialVersionUID", "serialPersistentFields", "IDENT"))
```

Comment:
I'm not sure why the name `IDENT` is ignored but it looks like an accident.. Since it's now configurable, users that need it can add it to the exclusion list.
```suggestion
                    .defaultValues("serialVersionUID", "serialPersistentFields")
```